deploy.sh

```diff
@@ -18,6 +18,8 @@ pip="$venv_dir/bin/pip"
 python_version="python3.12"
 python="$venv_dir/bin/python"
 env_file="$sort_dir/.env"
+frontend_dir="$sort_dir/assets/sort-survey-configurator"
+node_version=22
```

Comment:
Some dependencies were broken for v22 before so let's keep it at 20 for now until we can test it.

```suggestion
node_version=20
```